zhaquirks/tuya/tuya_smoke.py

```diff
@@ -81,6 +81,7 @@ def _update_attribute(self, attrid, value):
     .applies_to("_TZE284_rccxox8p", "TS0601")
     .applies_to("_TZE200_vzekyi4c", "TS0601")
     .applies_to("_TZE204_vawy74yh", "TS0601")
+    .applies_to("_TZE284_btsuytky", "TS0601")
```

Comment:
I think you can remove this change, the quirk builder below matches the device and exposes the same feature.